powerauth-restful-security-javaee/src/main/java/io/getlime/security/powerauth/rest/api/jaxrs/service/v2/ActivationService.java

```diff
@@ -87,7 +87,7 @@ public ActivationCreateResponse createActivation(ActivationCreateRequest request
 
             return response;
         } catch (Exception ex) {
-            logger.warn("Creating PowerAuth activation failed", ex);
+            logger.warn("Creating PowerAuth activation failed, error: {}", ex.getMessage());
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.